tensorflow/python/kernel_tests/transpose_op_test.py

```diff
@@ -388,6 +388,8 @@ def testComplex64(self):
     self._testBoth(
         np.complex(1, 2) *
         np.arange(0, 1260).reshape([2, 3, 5, 7, 2, 3]).astype(np.complex64))
+    self._testBoth(np.array(np.complex(1, 2)).astype(np.complex64))
```

Comment:
nit: put these above the others so they test in increasing rank order.

Reply:
Thanks @cantonios. Done.